src/components/material/infomedia/InfomediaModalBody.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { GetInfomediaQuery } from "../../../core/dbc-gateway/generated/graphql";
+
+export interface InfomediaModalBodyProps {
+  infomediaArticle: GetInfomediaQuery["infomedia"]["article"];
```

Comment:
Please use general types needed by the modal for props e.g. `title: string` and `text: ` instead of referencing the generated types.

This decouples the component.

Also it seems as if there are a lot of optional values involved in the article. It would be preferable if we ensured that we actually have what resembles an actual article (e.g. `text` must be set, perhaps `headline` is optional) before we consider it something worth rendering.